src/stylesheets/elements/form-controls/_checkbox-and-radio.scss

```
@@ -104,7 +114,13 @@
     &:checked,
     &:checked:disabled {
       @include format-input {
-        @include add-background-svg($input-checkmark);
+        @include add-background-svg("$input-checkmark");
```

No quotes are needed if you're passing a variable.

```suggestion
        @include add-background-svg($input-checkmark);
```

---

Resolved in https://github.com/uswds/uswds/pull/4509/commits/47d022094f7693c87bcaea6bcd55b3c111a03baf